attributes/default.rb

```diff
@@ -28,12 +28,11 @@
 default['elasticsearch']['routing_allocation_disk_watermark_low_threshold'] = "100gb"
 default['elasticsearch']['routing_allocation_disk_watermark_high_threshold'] = "50gb"
 default['elasticsearch']['routing_allocation_disk_watermark_flood_stage_threshold'] = "10gb"
-default['elasticsearch']['node_awareness_value'] = "$HOSTNAME"
-default['elasticsearch']['node_awareness_attribute'] = 'hostname'
 
 # ES 7.x
 default['elasticsearch']['initial_master_nodes'] = 'elasticsearch.service.consul'
 default['elasticsearch']['discovery_seed_hosts'] = 'elasticsearch.service.consul'
+default['elasticsearch']['discovery_type'] = 'multiple-node'
```

Comment:
so we will have to set `discovery_type` for small cluster on our barito, right? please note this or adjust directly on our cluster_templates.

Reply:
Correct, i follow Elasticsearch by setting the `discovery_type` as multiple-node by default. Single node cluster will need to set this value to `single-node`. Where should i put the documentation on this?

Reply:
we can start documented this on our jira. We can have explanation per each component or per cluster type.